### ISSUE TITLE
Pokemon Stadium 1 & 2: weird image shakiness and custom textures broken

### ISSUE THREAD

**Issue by overlordchaos** [gonetz/GLideN64#2761]
on latest public release 4.0 custom textures (i dumped the textures, run them trough an AI to resmaster them and upscale to 4x preserving the format and transparencies they where dumped with) prevoius versions did not have any issue. latest public release (4.0) adds to the main menu a shakiness so fast it makes me feel dizzy. plus many pokemons with custom textures with internal res of x4 are damaged. for example:
in some cases mew textures are half missing
in most cases mewtwo has two bellys
some pokemons have multiple mouths
in stadium 1 on VS Mewtwo stage that is a crystal arena flotating over the pokemon world clouds, the clouds get "stiff" when they must move around in random locations.
in stadium 2 most menu shadows are converted into semi transparent "glass" textures (should be a regular shadow but the custom textures are rendered as if it where pieces of glasses on top of the shadows)
in stadium 1 pokemon name, status and health menu on battle the custom borders instead of beeing all silver in some cases appear as dark brown.
one pokemon on stadium 2 (dont remember the name usually dont play with it) has 4 mouths where only one should be.
one mini game on stadium 2 where you cut tree logs at certain height to win, with custom textures the wooden log texture is replaced by a bright purple texture.
on stadium 1 and 2 pikachu and raichu, in most cases the textures are missing/messed up when using custom textures.
start screen of stadium 1 and 2 looks weird since its a single image/wallpaper divided into tiny cubes that once upscaled (original version looks like garbage TBQH) looks like currupted data in some cases since they look like paint strips instead of an actual wallpaper.
some different colored fire torches (can be orange, green red or blue depending on the stage) on stadium 1 and 2 sometimes appear with "fire" static effect but in the center you can see a static image of the "fire" as if it where a lightbulb with a fire shape on it with custom textures.
some menus on stadium 1 and 2 are messed up. for example on stadium 1 sometimes the hand that works as a pointer on menus dissapear.
main menu "shakiness" once it stops makes even the custom textures to look pixelated and blurry (its like if a blurry pixelated filter where applied to the game screen untill it goes 3D like in the stadium battles or mini games aka on 2D menus)

**Comment by overlordchaos** [comment]
> Public Release 4.0 is nearly four years old by this point. You should really be using the latest nightly instead. Download it [here](https://github.com/gonetz/GLideN64/actions/runs/4216187716).

yea tested stadium 1 and most issues are gone. still i have mewtwo with two bellys and articuno with 2 chests. leaving screenshots with original texture and custom (remastered and expanded x4 textures)
mewtwo original:
![Captura de pantalla 2023-02-22 170613](https://user-images.githubusercontent.com/18064513/220747923-c9079c00-4ce7-4b76-9d8e-4680f7e6d21f.png)
mewtwo "remaster" textures
![Captura de pantalla 2023-02-22 170235](https://user-images.githubusercontent.com/18064513/220748027-195e1b2e-f44d-4436-bd49-84345bca4de6.png)
articuno original:
![Captura de pantalla 2023-02-22 170856](https://user-images.githubusercontent.com/18064513/220748146-65b9b874-b1df-4a1a-ad6a-64de5112b71f.png)
articuno "remaster" textures:
![Captura de pantalla 2023-02-22 170826](https://user-images.githubusercontent.com/18064513/220748251-29b2cb4a-0413-4df8-b213-351d0b2da1e0.png)
still results are more accurate and better than public release.

**Comment by overlordchaos** [comment]
just to report that with the latest builds many many pokemons have texture issues on pokemon stadium 1. and in stadium 2 some pokemons and some stages.

**Comment by DonelBueno** [comment]
@overlordchaos do you have per-pixel lighting enabled for this game? This setting causes issues on pokémon shadowing.

![image](https://user-images.githubusercontent.com/4239694/232408840-ea9b72e7-3874-421e-a012-5903b73cfb19.png)

**Comment by overlordchaos** [comment]
> @overlordchaos do you have per-pixel lighting enabled for this game? This setting causes issues on pokémon shadowing.
> 
> ![image](https://user-images.githubusercontent.com/4239694/232408840-ea9b72e7-3874-421e-a012-5903b73cfb19.png)

yes. disabled it but cannot test it. deleted pokemons stadiums since i came to realize that A ash was retired from the show again, and B the whole franchise is about magical animals hurting themselves untill faint or in some cases death. and i hate animal cruelty. so someone else must test this since roms, save files and custom textures are gone.
